core/blocktree/blocktree_test.go

```diff
@@ -38,24 +36,12 @@ func createGenesisBlock() types.Block {
 		},
 		Body: &types.BlockBody{},
 	}
-	b.Header.SetHash(common.Hash{0x00})
+	_, _ = b.Header.Hash()
```

Comment:
Why call this here?

Reply:
you mean the underscore part? I can remove